homeassistant/components/shelly/light.py

```diff
@@ -48,21 +65,41 @@ def brightness(self):
         return int(brightness / 100 * 255)
 
     @property
-    def supported_features(self):
-        """Supported features."""
-        return self._supported_features
+    def color_temp(self) -> Optional[float]:
+        """Return the CT color value in mireds."""
+        if self.control_result:
+            color_temp = self.control_result["colorTemp"]
+        else:
+            color_temp = self.block.colorTemp
+        return int(color_temperature_kelvin_to_mired(color_temp))
+
+    @property
+    def min_mireds(self) -> float:
+        """Return the coldest color_temp that this light supports."""
+        return color_temperature_kelvin_to_mired(2700)
+
+    @property
+    def max_mireds(self) -> float:
+        """Return the warmest color_temp that this light supports."""
+        return color_temperature_kelvin_to_mired(6500)
 
     async def async_turn_on(
-        self, brightness=None, **kwargs
-    ):  # pylint: disable=arguments-differ
+        self, brightness=None, color_temp=None, **kwargs
```

Comment:
can I add `color_temp` here? or should I get `brightness` and `color_temp` from kwargs, like in hive or esphome integrations?

Reply:
I've changed `async_turn_on` to look the same as in hive and esphome